platform/mds/mds-web/src/main/resources/webapp/js/controllers.js

```diff
@@ -2854,26 +2859,29 @@
         * the entity in situation in which the entity was selected from the entity list.
         */
         $scope.$watch('selectedEntity', function () {
-            blockUI();
+            $scope.selectedEntityChanged();
+        });
 
+        $scope.selectedEntityChanged = function() {
             if ($scope.selectedEntity && $scope.selectedEntity.id) {
-                workInProgress.setActualEntity(Entities, $scope.selectedEntity.id);
-
-                $scope.fields = Entities.getFields({id: $scope.selectedEntity.id}, function () {
-                    setSecuritySettings();
-                    setAdvancedSettings();
-                    $scope.draft({});
-                });
-
-                unblockUI();
+                if (workInProgress.actualEntity !== $scope.selectedEntity.id) {
+                    blockUI();
+                    workInProgress.setActualEntity(Entities, $scope.selectedEntity.id);
+
+                    $scope.fields = Entities.getFields({id: $scope.selectedEntity.id}, function () {
+                        setSecuritySettings();
+                        setAdvancedSettings();
```

Comment:
seems that after these changes, the abandon changes button does not work properly